### ISSUE TITLE
Marketing and New Updates

### ISSUE THREAD

**Issue by leonard-dunsey** [Yink/Amadeus#99]
the app has a great interface, in addition to great features and functionality,
the app has a future, but what remains for it to proceed are two things:
**- more dissemination:**
if the app were more publicized, if it addressed a specific audience, it would certainly have more "audience".
**- more updates:**
the last update was last year, that is, how do you want to keep the app standing?
although the app is excellent, there are still some things to improve.
### I hope they don't abandon the project, because the idea is very good, and very i''m fan of anime Steins;Gate, and i belive than i'm not alone

**Comment by NikaVor** [comment]
не одинок

**Comment by lnfel** [comment]
The last commit was 2 years ago tho, and no announcement or update ever since.
But hey, **I remembered this awesome project and decided to clone the repo today**.
I have not a single experience working with Java but I'll see what I can do.
It might be a slow pace since I'm working on a different project atm.
If anyone interested in helping out just ping me here.

**Comment by jeluchu** [comment]
@lnfel I would like to help you, a little time ago I tried to migrate the entire project to a more current language such as Kotlin, in addition to updating all the libraries and migrating everything to the most updated Android components (AndroidX), I am in various projects of mine, but I can always help in some free moments (I am an Android developer and I love Steins; Gate)

**Comment by lnfel** [comment]
@Jeluchu that's awesome! First we need to setup a repo, if your migration to AndroidX is successful then you can host the repo for the stable version and add me as collaborator so I would be notified. I did what I could last Saturday but I got stuck with an Error upon migrating to AndroidX.

**Comment by jeluchu** [comment]
@lnfel I have already managed to migrate the project to AndroidX without any error and update both the Gradle and any library that is inside the project. The only incident that I have detected (or it is the mobile in which I use it) is that when I speak it does not finish recognizing the command, but that can be tested, it would also be good to migrate the code to Kotlin

**Comment by lnfel** [comment]
@Jeluchu great! I just checked your [Amadeus](https://github.com/Yink/Amadeus) repo. I'll try cloning that on my free time and see if I can make it work on my environment. 

> The only incident that I have detected (or it is the mobile in which I use it) is that when I speak it does not finish recognizing the > command, but that can be tested, it would also be good to migrate the code to Kotlin

I was thinking of re-writing the code base as well.
Maybe we can talk this over with a messaging app sometime so we could discuss future plans of the project. ✌️

**Comment by jeluchu** [comment]
@lnfel Yes, although the repo you have seen I have to upload the changes that I made today, on that occasion I made some changes but since then Android has made more changes, so if that this afternoon I will try to update the code to upload it to the repo. 

> I was thinking of re-writing the code base as well.
> Maybe we can talk this over with a messaging app sometime so we could discuss future plans of the project. ✌️

One of the interesting changes to what you mean by changing the base code is the optimization, refactoring, and change in the code language, in addition to being able to implement a possible architecture. I will upload the new changes for today and if I have time I try to migrate the code this afternoon since I am currently finishing preparing an update of an anime app that I have

**Comment by lnfel** [comment]
> One of the interesting changes to what you mean by changing the base code is the optimization, refactoring, and change in the code language, in addition to being able to implement a possible architecture. I will upload the new changes for today and if I have time I try to migrate the code this afternoon since I am currently finishing preparing an update of an anime app that I have

@Jeluchu nice, we're on the same page.
